WWW/2_linear_chain.wdl

```diff
@@ -13,9 +13,9 @@ version 1.0
 
 workflow minidata_test_alignment {
   input {
-    # Batch and cohort information
+    # Sample info
     File sampleFastq
-    String base_file_name = "SRR8618962_combined"
+    String base_file_name
```

Comment:
Whichever option we choose for base_file_name for the first workflow, we should also keep consistent here.